File: Services/Public/Public.Domain/Review.cs
using System.ComponentModel.DataAnnotations;

namespace Public.Domain;

public class Review
{
    public int review_id { get; set; }
    [Required]
    public string title { get; set; }
    [Required]
    public int writer_id { get; set; }
    [Required]
    public int scam_type_id { get; set; }
    [Required]
    public DateTime review_date { get; set; }
    [Required]
    public int review_content_id { get; set; }
    public int? views { get; set; }
    
}

File: Services/DB_Handler/models/Review.py
from sqlalchemy import Column, Integer, String, Date, ForeignKey
from sqlalchemy.orm import relationship
from .base import Base


class Review(Base):
    __tablename__ = 'review'

    review_id = Column(Integer, primary_key=True, nullable=False)
    title = Column(String(50), nullable=False)
    writer_id = Column(Integer, ForeignKey("admins.admin_id"))
    scam_type_id = Column(Integer, ForeignKey("scam_type.scam_type_id"))
    review_date = Column(Date, nullable=False)
    review_content_id = Column(Integer, nullable=False)
    views = Column(Integer, default=0)

